rust/sdk-processor/src/utils/starting_version.rs

```diff
@@ -42,6 +42,77 @@ pub async fn get_starting_version(
     ))
 }
 
+pub async fn get_minimum_last_success_version(
```

Comment:
Let's add a docstring here + add append `parquet` to the end of the function name oslt. What do you think? I'm worried that colocating these functions w/ the other postgres ones may be confusing.